tools/flang1/flang1exe/main.c

```diff
@@ -761,6 +763,7 @@ init(int argc, char *argv[])
   register_boolean_arg(arg_parser, "es", (bool *)&(flg.es), false);
   register_boolean_arg(arg_parser, "pp", (bool *)&(flg.p), false);
 
+  register_boolean_arg(arg_parser, "list-macros", &list_macros, false);
   /* Set values form command line arguments */
```

Comment:
add one empty line in between. The author of the code below wanted it to be separated from the `register...` part and we should obey that.